package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "uploadcare-redactor",
+  "version": "1.2.0",
+  "description": "This is Uploadcare plugin for [Imperavi Redactor] text editor. It will allow your users to upload files and images from local device, social networks, cloud storages without any backend code that is usually required to handle uploads.",
+  "main": "uploadcare.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/TimonVS/uploadcare-redactor.git"
+  },
+  "author": "Uploadcare",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/uploadcare/uploadcare-redactor/issues"
```

Comment:
can we have several urls here?
so users could file redactor related issues here and widget related issues [there](https://github.com/uploadcare/uploadcare-widget/issues/)